modal/functions.py

```diff
@@ -292,7 +292,10 @@ async def _map_invocation(
     have_all_inputs = False
     num_inputs = 0
     num_outputs = 0
-    pending_outputs = {}  # map input_id -> next expected gen_index value
+    # Map input_id -> next expected gen_index value, or -1 if complete
```

Comment:
Where is the code that sets it to -1 if complete? Earlier version of the PR?

Reply:
Fixing comment! Thanks for catching